### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by CMUgarte**
Also standard prep for the release. We are just waiting on the response from @LinguList to see whether we close issue #13 or not for the lexibank release.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -13,11 +13,11 @@
     "contributors": [
         {
             "name": "Johann-Mattis List",
-            "type": "Other"
+            "type": "Editor"
         },
         {
             "name": "Mei-Shin Wu",
-            "type": "Other"
+            "type": "Editor"
```
Comment (FredericBlum) [inline .zenodo.json:20]: Mei-Shin is not an editor on this

**Review by FredericBlum — CHANGES_REQUESTED** [review]

File: .zenodo.json (hunk 1 above)
Comment (MuffinLinwist) [reply .zenodo.json:20]: you are right, probably `Other` would be a better fit on this case. Can I have confirmation on this from you, @LinguList, please?

Comment (chrzyki) [reply .zenodo.json:20]: Agreed with @FredericBlum

**Review by chrzyki — CHANGES_REQUESTED** [review]

**Comment by MuffinLinwist** [comment]
I adopted the requested changes in `CONTRIBUTORS.md`, @FredericBlum and @chrzyki.

**Review by FredericBlum — APPROVED** [review]

**Review by chrzyki — APPROVED** [review]

**Pull request merged**